public/video-ui/src/reducers/rootReducer.js

```diff
@@ -1,11 +1,13 @@
 import { combineReducers } from 'redux';
+import config from './configReducer';
 import error from './errorReducer';
 import video from './videoReducer';
 import videos from './videosReducer';
 import { reducer as form } from 'redux-form'
 
 
 export default combineReducers({
```

Comment:
Kind of coincidence, I unconsciously did it as that's the order they're in the folder in.